src/objects/Facility__c.object

```diff
@@ -0,0 +1,189 @@
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Parent_Facility__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>Lets you identify a facility as part of a larger one (for example, a building might be a parent of a classroom).</description>
+        <externalId>false</externalId>
+        <inlineHelpText>Lets you identify a facility as part of a larger one (for example, a building might be a parent of a classroom).</inlineHelpText>
+        <label>Parent Facility</label>
+        <referenceTo>Facility__c</referenceTo>
+        <relationshipLabel>Facilities</relationshipLabel>
+        <relationshipName>Facilities</relationshipName>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <fields>
+        <fullName>Type__c</fullName>
+        <description>The type of facility.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The type of facility.</inlineHelpText>
+        <label>Type</label>
+        <picklist>
+            <picklistValues>
+                <fullName>Building</fullName>
+                <default>false</default>
+            </picklistValues>
+            <picklistValues>
+                <fullName>Classroom</fullName>
+                <default>false</default>
+            </picklistValues>
+            <picklistValues>
+                <fullName>Computer Lab</fullName>
+                <default>false</default>
+            </picklistValues>
+            <picklistValues>
+                <fullName>Food Hall/Cafeteria</fullName>
+                <default>false</default>
+            </picklistValues>
+            <picklistValues>
+                <fullName>Health Center</fullName>
+                <default>false</default>
+            </picklistValues>
+            <picklistValues>
+                <fullName>Gymnasium</fullName>
+                <default>false</default>
+            </picklistValues>
+            <picklistValues>
+                <fullName>Laboratory</fullName>
+                <default>false</default>
+            </picklistValues>
+            <picklistValues>
+                <fullName>Lecture Hall</fullName>
+                <default>false</default>
+            </picklistValues>
+            <picklistValues>
+                <fullName>Library</fullName>
+                <default>false</default>
+            </picklistValues>
+            <restrictedPicklist>true</restrictedPicklist>
+            <sorted>false</sorted>
+        </picklist>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+    </fields>
+    <label>Facility</label>
+    <listViews>
+        <fullName>All</fullName>
+        <filterScope>Everything</filterScope>
+        <label>All</label>
+    </listViews>
+    <nameField>
+        <label>Facility Name</label>
```

Comment:
@lifewithryan I don't quite remember why we landed on doing it this way, but I'm wondering why I didn't explore the option of just having this be an autonumber. There could be a separate "Facility Name" field that would be something like "Room 201". The admin could decide to create a formula field that displays a more friendly name based on the Account or Parent Facility. This could negate our need for custom facility naming logic. Perhaps search would be affected by this? (Perhaps it's too late to go down this path...)

Account naming makes sense because it can't be achieved through a formula since its many contacts to an account. But this is a one-to-one relationship between a Facility and either a Parent Facility or an Account...

Reply:
This is a very good point. So are you suggesting we have something like FAC-1021 and then we perhaps "ProTip" their friendly name? They could then add it to the fields that show in the search view?

I think I like this idea, but before we jump on it, we should ask @aligeez and @JasonFitzy. We can always add code later.

Reply:
I mentioned this in our HEDA chat so we can have a bigger discussion. :)

Reply:
Update here... I left this as-is as we discussed earlier today. We're going to add in an unmanaged formula field in #720